scripts/quests/windurst/All_At_Sea.lua

```diff
@@ -0,0 +1,168 @@
+-----------------------------------
+-- All at Sea
+-----------------------------------
+-- !addquest 2 23
+-- Paytah !gotoid 17760350
+-- Baren-Moren !gotoid 17752232
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.ALL_AT_SEA)
+
+quest.reward =
+{
+    item = xi.item.LEATHER_RING,
+    fame = 30,
+    fameArea = xi.quest.fame_area.WINDURST,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getFameLevel(xi.quest.fame_area.WINDURST) >= 3
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Paytah'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(291)
+                end,
+
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.item.RIPPED_CAP) then
+                        return quest:progressEvent(292)
+                    end
+                end,
```

Comment:
Can you really start the quest directly by trading without speaking to the NPC?

Reply:
Good point, speaking to Paytah does not initiate the quest but I've put in a capture request to confirm you don't need to speak with him at all

Reply:
https://www.youtube.com/watch?v=NyrhHhkVydk

Confirmed by capture, you can indeed start the quest with a trade and without speaking to NPC